crates/sema/src/ast_passes.rs

```diff
@@ -105,6 +106,15 @@ impl<'ast> Visit<'ast> for AstValidator<'_> {
                         .emit();
                 }
             }
+            StmtKind::UncheckedBlock(block) => {
+                if self.in_unchecked_block {
+                    self.dcx().err("\"unchecked\" blocks cannot be nested.").span(stmt.span).emit();
```

Comment:
we should use '`' (backticks) for keywords and code in messages

Reply:
There's no need for compatibility in this case, and yes please also change that one too in this PR

Reply:
Done